src/swagger/api.yaml

```diff
@@ -7546,6 +7546,51 @@ definitions:
         template_swa: '#/definitions/SwaApplication'
         template_swa3field: '#/definitions/SwaThreeFieldApplication'
       propertyName: name
+  CatalogApplication:
+    properties:
+      id:
+        type: string
+        readOnly: true
+      name:
+        type: string
+      displayName:
+        type: string
+      description:
+        type: string
+      status:
+        $ref: '#/definitions/CatalogApplicationStatus'
+      lastUpdated:
+        format: date-time
+        readOnly: true
+        type: string
+      category:
+        type: string
+      verificationStatus:
+        type: string
+      website:
+        type: string
+      signOnModes:
+        items:
+          type: string
+        type: array
+      features:
+        items:
+          type: string
+        type: array
+      _links:
+        additionalProperties:
+          type: object
+        readOnly: true
+        type: object
+    x-okta-tags:
+      - Role
```

Comment:
Is this right? `Role`?

Reply:
The corresponding method is tagged as `user`

Reply:
cc: @bretterer

Reply:
@bdemers raised this question with @bretterer in his original PR. 

Here is his feedback - https://github.com/okta/openapi/pull/243#discussion_r440904222

Reply:
This is an Administrator Role model, which is why its tagged as `Role`

Reply:
`CatalogApplication` is an Administrator Role?

Oh I see now, it's from:
> `/api/v1/groups/{groupId}/roles/{roleId}/targets/catalog/apps`

I think this object might get confused with the other `*Application` objects.  But... tagging it with `role` (which will namespace it in the Java SDK), should remove that confusion

I withdraw my objection 🧑‍⚖️

Reply:
Thanks @bretterer & @bdemers !